### PR TITLE
[MOBILE] Fixing Sample Crash Issue

### PR DESCRIPTION
**Pull request by d1-agarwal**
Updating Xamarin Form version to avoid sample app crash issue.

### CODE REVIEW

File: Mobile/Accounts/SampleAccount/SampleAccount.Tizen.Mobile/SampleAccount.Tizen.Mobile.csproj [hunk 1]
```diff
@@ -27,11 +27,11 @@
...
   <!-- Include Nuget Package for Tizen Project building -->
   <ItemGroup>
-    <PackageReference Include="Tizen.NET" Version="4.0.0-preview1-00317">
+    <PackageReference Include="Tizen.NET" Version="5.0.0.14629">
       <ExcludeAssets>Runtime</ExcludeAssets>
     </PackageReference>
-    <PackageReference Include="Tizen.NET.Sdk" Version="1.0.0-pre4" />
-    <PackageReference Include="Xamarin.Forms.Platform.Tizen" Version="2.4.0-r282-001" />
+    <PackageReference Include="Tizen.NET.Sdk" Version="1.0.1" />
+    <PackageReference Include="Xamarin.Forms.Platform.Tizen" Version="2.5.1.444934" />
```
Comment (myroot) [inline Mobile/Accounts/SampleAccount/SampleAccount.Tizen.Mobile/SampleAccount.Tizen.Mobile.csproj]: You can remove it

Comment (d1-agarwal) [reply Mobile/Accounts/SampleAccount/SampleAccount.Tizen.Mobile/SampleAccount.Tizen.Mobile.csproj]: Is this package
Xamarin.Forms.Platform.Tizen
not reqiured ?

Comment (myroot) [reply Mobile/Accounts/SampleAccount/SampleAccount.Tizen.Mobile/SampleAccount.Tizen.Mobile.csproj]: Yes, no need anymore

Comment (d1-agarwal) [reply Mobile/Accounts/SampleAccount/SampleAccount.Tizen.Mobile/SampleAccount.Tizen.Mobile.csproj]: Okay, Thanks.

**Pull request merged**